src/General-Rules-Tests/ReAsClassRuleTest.class.st

```diff
@@ -0,0 +1,32 @@
+Class {
+	#name : 'ReAsClassRuleTest',
+	#superclass : 'ReAbstractRuleTestCase',
+	#classInstVars : [
+		'critiques'
```

Comment:
This instance variable is unused and will make a release test checking that all instance variables are used fail.